test/e2e/apimachinery/crd_validation_rules.go

```diff
@@ -35,7 +35,7 @@ import (
 	admissionapi "k8s.io/pod-security-admission/api"
 )
 
-var _ = SIGDescribe("CustomResourceValidationRules [Privileged:ClusterAdmin][Alpha][Feature:CustomResourceValidationExpressions]", func() {
```

Comment:
if is still beta you have to keep the Feature tag, there are some jobs that runs only GA feature, disabling the betas, if they pick this test those will fail

Reply:
hmm, I'm not sure about this, maybe is something I'm making up or mixing with other things,  @BenTheElder can you confirm if is ok to remove the Feature tag here?
if we keep the Feature tag, these tests will not run on presubmits :/

Reply:
I guess with this PR(https://github.com/kubernetes/test-infra/pull/26994) merged. The feature could be added back.

Reply:
> if is still beta you have to keep the Feature tag, there are some jobs that runs only GA feature, disabling the betas, if they pick this test those will fail

actually, the GA-only job only runs conformance tests since those are supposed to be guaranteed to work. other tests cannot be.

I think we can drop [Feature], Feature is for things that may not reasonably be available in the cluster like Alpha, or extensions like loadbalancer implementation

we have a KEP pending to clarify this a bit https://github.com/kubernetes/enhancements/pull/3042